loggroup-lambda-connector/test/loggroup-lambda-cft.json

```diff
@@ -0,0 +1,186 @@
+{
+    "AWSTemplateFormatVersion": "2010-09-09",
+    "Outputs": {
+        "SumoLogGroupLambdaConnector": {
+            "Description": "SumoLogGroupLambdaConnector Function ARN",
+            "Value": {
+                "Fn::GetAtt": [
+                    "SumoLogGroupLambdaConnector",
+                    "Arn"
+                ]
+            }
+        }
+    },
+    "Resources": {
+        "SumoLogGroupLambdaConnector": {
+            "Type": "AWS::Lambda::Function",
+            "Properties": {
+                "Code": {
+                    "S3Bucket": "appdevstore",
```

Comment:
I created a bucket called "sumologic-appdevstore". You can use this to make it look more official

Reply:
Yes agreed let's migrate rest of the functions as well in separate PR